tests/foreman/virtwho/conftest.py

```diff
@@ -0,0 +1,105 @@
+import logging
```

Comment:
I see this file is copied from `tests/foreman/ui/conftest.py`, with some parts removed. I wonder if there is a way for us to share these fixtures between two directories.
If nobody has any good idea, we can deal with that duplication later.

Reply:
We can use `sys.path.append('../')` to import the conftest.py from ui, but I think it's not a good idea.
about the @stubbed, yes, I have add some doc string cases in virthwo, these cases should be automated in the future, it will not finished in this PR.